libraries/protocol/include/graphene/protocol/chain_parameters.hpp

```diff
@@ -50,6 +50,9 @@ namespace graphene { namespace protocol {
       const fee_schedule& get_current_fees() const { FC_ASSERT(current_fees); return *current_fees; }
       fee_schedule& get_mutable_fees() { FC_ASSERT(current_fees); return const_cast<fee_schedule&>(*current_fees); }
 
+      //==================================================================================
+      double                  my_term                             = 587.024;
```

Comment:
We don't use `float` nor `double` in consensus, because it is usually inaccurate. See https://stackoverflow.com/questions/3730019/why-not-use-double-or-float-to-represent-currency.